src/index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>LEADERBOARD</title>
+</head>
+
+<body>
+  <main class="container">
+    <h1 class="main-title">LEADERBOARD</h1>
+    <div class="main-area">
+      <div class="results">
+        <div class="result-header">
+          <h2>Recent scores</h2>
+          <button type="button" class="btn">Refresh</button>
+        </div>
+        <ul class="result-list">
+          <li>Name: 100</li>
+          <li>Name: 20</li>
+          <li>Name: 50</li>
+          <li>Name: 78</li>
+          <li>Name: 125</li>
+          <li>Name: 77</li>
+          <li>Name: 42</li>
+        </ul>
+      </div>
+      <div class="entries">
+        <h2 class="entries-title">Add yor score</h2>
+        <form action="/" class="entries-form">
+          <input type="text" id="name" name="name" placeholder="Your Name" required
+            class="form-control" />
+          <input type="number" id="score" name="score" placeholder="Your Score" required
+          class="form-control" />
+          <div class="form-submit">
+            <button type="submit" class="btn" >Submit</button>
+          </div>
+        </form>
+      </div>
+    </div>
+
+  </main>
+</body>
+
+</html>
```

Comment:
[OPTIONAL] Always remember to be careful with your indentation. This one it's important as it keeps the code readable and organized, which is very valuable, especially when working in a team or open-source projects.

Reply:
Thank you for your suggestions, Daniel 👍 .